helm/README.md

```diff
@@ -136,74 +132,70 @@ The following tables lists the configurable parameters of the NS1 DDI chart and
+| `dhcp.pops[#].resources` | CPU/memory resource requests/limits. | `{}` |
+| `dhcp.pops[#].nodeSelector` | Node labels for pod assignment. | `{}` |
+| `dhcp.pops[#].tolerations` | Node tolerations for pod assignment. | `[]` |
+| `dhcp.pops[#].affinity` | Node affinity for pod assignment. | `{}` |
 
 ### XFR
-By default, XFR is disabled (i.e. set to `{}` in `values.yaml`).
-
-| Parameter | Description | Default |
+| Parameter | Description | Example |
 | --------- | ----------- | ------- |
-| `xfr.name` | Name to be used for the various XFR resources. | |
-| `xfr.replicas` | Number of XFR replicas to run. | |
-| `xfr.image.name` | The name of the image to use for the XFR container. | |
-| `xfr.image.tag` | The tag of the image to use for the XFR container. | |
-| `xfr.image.pullPolicy` | The pull policy for the image. | |
-| `xfr.livenessProbe.initialDelaySeconds` | How long to wait for the XFR pods to come up before beginning health checks. | |
-| `xfr.livenessProbe.failureThreshold` | How many failed healthchecks are tolerated prior to restarting the pod. | |
-| `xfr.livenessProbe.periodSeconds` | How often to execute healthchecks. | |
-| `xfr.popID` | The ID of the PoP. | |
-| `xfr.coreService` | The name of the Kubernetes service for core. | |
-| `xfr.enableOpsMetrics` | Enables operational metrics. | |
-| `xfr.startupFlags` | Additional flags to pass to the startup command of the XFR container. | |
-| `xfr.resources` | CPU/memory resource requests/limits. | |
-| `xfr.nodeSelector` | Node labels for pod assignment. | |
-| `xfr.tolerations` | Node tolerations for pod assignment. | |
-| `xfr.affinity` | Node affinity for pod assignment. | |
+| `xfr.name` | Name to be used for the various XFR resources. | `xfr` |
+| `xfr.replicas` | Number of XFR replicas to run. | `1` |
+| `xfr.image.name` | The name of the image to use for the XFR container. | `ns1inc/privatedns_xfr` |
+| `xfr.image.tag` | The tag of the image to use for the XFR container. | `3.3.3` |
+| `xfr.image.pullPolicy` | The pull policy for the image. | `IfNotPresent` |
+| `xfr.livenessProbe.initialDelaySeconds` | How long to wait for the XFR pods to come up before beginning health checks. | `30` |
+| `xfr.livenessProbe.failureThreshold` | How many failed healthchecks are tolerated prior to restarting the pod. | `3` |
+| `xfr.livenessProbe.periodSeconds` | How often to execute healthchecks. | `15` |
+| `xfr.popID` | The ID of the PoP. | `default_pop` |
+| `xfr.coreService` | The name of the Kubernetes service for core. | `core` |
+| `xfr.enableOpsMetrics` | Enables operational metrics. | `false` |
+| `xfr.startupFlags` | Additional flags to pass to the startup command of the XFR container. | `{}` |
+| `xfr.resources` | CPU/memory resource requests/limits. | `{}` |
+| `xfr.nodeSelector` | Node labels for pod assignment. | `{}` |
+| `xfr.tolerations` | Node tolerations for pod assignment. | `[]` |
+| `xfr.affinity` | Node affinity for pod assignment. | `{}` |
```

Comment:
I think this looks good to me, maybe something in the readME about the core and data service definitions, and having to set
`--service_def_id <int>` if not using the bootstrap wizard